.github/workflows/snyk-issue.yml

```diff
@@ -0,0 +1,29 @@
+name: Snyk Issue
```

Comment:
This file sounds not related to this PR.

Reply:
OK